File: Level 064/homework/hw9.js
// 9)შემოიტანე 1 input და 1 ცლადი შეადარე, არის თუ არა ერთი რიცხვი მეორეზე ნაკლები ან ტოლი.

let input = prompt("Enter num: ")
let input1 = prompt("Enter second num: ")

if (input < input1) {
    console.log("ნაკლებია")
}

else if (input > input1) {
    console.log("მეტია")
}

else {
    console.log("ტოლია")
}

File: Level 008/homework/hw9.py
# დაწერე პროგრამა, რომელიც ამოწმებს, არის თუ არა რიცხვი ლუწი ან 3-ის ჯერადი.

num = int(input("Enter num: "))

if num % 2 == 0:
    print("even")
elif num % 3 == 0:
    print("ჯერადია")
else:
    print("არცერთი არ არის")